src/domain/_common/components/VariantButton.tsx

```diff
@@ -15,12 +16,12 @@ const buttonStyle = {
   large: 'text-white bg-red-default w-full min-w-[336px] h-11 active:bg-red-touch disabled:bg-red-disabled',
 };
 
-export function Button({ type, isDisabled, onClick, children }: React.PropsWithChildren<ButtonProps>) {
+export function VariantButton({ style, isDisabled, children, type = 'submit' }: React.PropsWithChildren<ButtonProps>) {
```

Comment:
```suggestion
export function VariantButton({ style, disabled=false, children, type = 'submit' }: React.PropsWithChildren<ButtonProps>) {
```
여기도 같이 바뀌어야.....
그리고 저 밑에도....